doc/man3/OSSL_CRYPTO_load_u16_le.pod

```diff
@@ -0,0 +1,83 @@
+=pod
+
+=head1 NAME
+
+OSSL_CRYPTO_load_u16_le, OSSL_CRYPTO_load_u16_be, OSSL_CRYPTO_load_u32_le,
+OSSL_CRYPTO_load_u32_be, OSSL_CRYPTO_load_u64_le, OSSL_CRYPTO_load_u64_be,
+OSSL_CRYPTO_store_u16_le, OSSL_CRYPTO_store_u16_be,
+OSSL_CRYPTO_store_u32_le, OSSL_CRYPTO_store_u32_be,
+OSSL_CRYPTO_store_u64_le, OSSL_CRYPTO_store_u64_be -
+Read and write unsigned 16, 32 and 64-bit integers in a specific byte order
+
+=head1 SYNOPSIS
+
+    #include <openssl/byteorder.h>
+
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u16_le(
+        uint8_t *out, uint16_t val);
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u16_be(
+        uint8_t *out, uint16_t val);
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u32_le(
+        uint8_t *out, uint32_t val);
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u32_be(
+        uint8_t *out, uint32_t val);
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u64_le(
+        uint8_t *out, uint64_t val);
+    static ossl_inline uint8_t *OSSL_CRYPTO_store_u64_be(
+        uint8_t *out, uint64_t val);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u16_le(
+        uint16_t *val, const uint8_t *in);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u16_be(
+        uint16_t *val, const uint8_t *in);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u32_le(
+        uint32_t *val, const uint8_t *in);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u32_be(
+        uint32_t *val, const uint8_t *in);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u64_le(
+        uint64_t *val, const uint8_t *in);
+    static ossl_inline const uint8_t *OSSL_CRYPTO_load_u64_be(
+        uint64_t *val, const uint8_t *in);
+
+=head1 DESCRIPTION
+
+These functions read and write 16, 32 and 64 bit unsigned integers in a
+specified byte order.
+The I<_be> functions use big-endian byte order, while the I<_le> functions
```

Comment:
I would use `C<_be>`, `C<_le>` as `I<>` should denote arguments.

Reply:
Ditto.